sdv/multi_table/hma.py

```diff
@@ -159,6 +160,7 @@ def __init__(self, metadata, locales=['en_US'], verbose=True):
         self._augmented_tables = []
         self._learned_relationships = 0
         self._default_parameters = {}
+        self.parent_extended_columns = defaultdict(list)
```

Comment:
we can probably make this private

Reply:
Done:
https://github.com/sdv-dev/SDV/pull/2068/commits/0490d81af6463be113af32a8b7f6cfeb80bd3fcd#diff-c18ffdc4352bb1e367eaefa2e851a0da762907b7bf727f00bc6094446c889beeR163